src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1099468813&format=interactive",
+  sheets_gid="1827909463",
+  sql_file="video_attribute_count_mobile.sql"
+  )
+}}
+
+#### Preload
+
+The most commonly used attribute is preload. The preload attribute gives the browser a hint on the best way to handle the video download. There are four possible options: `auto`, `metadata`, `none`, and an empty response = `auto`.
+
+{{ figure_markup(
+  image="video-preload-values.png",
+  caption="Video preload values.",
+  description="A bar chart showing the prevalence of the various preload attribute values, on desktop and mobile, in both 2020 and 2021. `auto` was used in 36% of preload attributes in 2020 on desktop, falling to 23% in 2021. On mobile, `auto` fell from 27% to 18%. `metadata` was used in 24% of preload attribuets on desktop in 2020, rising slightly to 25% in 2021. On mobile, `metadata` usage rose from 33% to 36%. The `none` value was present in 33% of preload attributes on desktop in 2020, rising to 46% in 2021. On mobile, usage also rose, from 33% to 40%. Finally, around 4% of preload attributes were present but left blank on desktop, and 5% on mobile, with no change for either between 2020 and 2021.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1322480946&format=interactive",
+  sheets_gid="7133313",
+  sql_file="video_attribute_count_mobile.sql"
+  )
+}}
+
+Interestingly, we see a large push away from `preload=auto` on both mobile and desktop, with most of the movement to `preload=none.`  While it is possible that this changed for many videos, it could just be that the new videos added to the web over the last year utilize the "none" parameter more than in the past. From a page weight perspective—this is a large win for the web.
+
+#### Autoplay
+
+The next most commonly used attribute is "autoplay."  This tells the browser that the video should play as soon as possible (and because of this, autoplay will actually override the preload attribute).
+
+The autoplay attribute is a boolean attribute, meaning that its presence by fault means true. So for the 190 sites that use `autoplay="false"`, we're sorry to tell you that is not going to work.
+
+#### Width
+
+The width attribute is also one of the top video attributes. It tells the browser how wide the video player should be (note that height is very rarely used, since the aspect ratio of the video will decide the space with just one value.)
```

Comment:
```suggestion
The `width` attribute is also one of the top video attributes. It tells the browser how wide the video player should be (note that `height` is very rarely used, since the aspect ratio of the video will decide the space with just one value.)
```

Actually I discovered recently that this is not the case! And unless you override with CSS a default aspect-ration of 2:1 will be used: https://github.com/whatwg/html/issues/3090

Reply:
Thoughts, @dougsillars?

Reply:
looks good to me

Reply:
@dougsillars We should absolutely use ticks around the attribute names, but I think @tunetheweb was calling the accuracy of your statement about aspect ratios into question. If they do indeed get a default aspect ratio, rather then filling out their natural aspect ratio, you should re-write this.